### PR TITLE
Merge dev-de in main

### PR DESCRIPTION
**Pull request by iamNoah1**
### Describe your changes
In order to get the completed terms online, we want to merge dev-de into main

### CODE REVIEW

**Review by seokho-son — APPROVED** [review]

File: config.toml [hunk 1]
```diff
@@ -68,6 +68,7 @@ description = "The CNCF Cloud Native Glossary Project is intended to be used as
 #time_format_default = "02.01.2006"
 #time_format_blog = "02.01.2006"
 
+
```
Comment (seokho-son) [inline config.toml:71]: We don't have to modify this part, but I guess it was from previous manual rebase.

In order to minimize mixed up situation, we'd better **update dev-de first**, and then, let's merge this PR.

**Comment by iamNoah1** [comment]
@jihoon-seo @seokho-son I guess we can proceed here?!

**Comment by seokho-son** [comment]
Hi @iamNoah1 :)

I had hoped that https://github.com/cncf/glossary/pull/2422 would help discard the minor change in the config.toml. However, it didn't work as expected. I suspect the change originated from a previous rebase of the dev-de branch.

We could potentially discard the change by altering the commit history, but the hassle might outweigh the benefits. 
I'll go ahead and merge this PR as is.

**Review by seokho-son — APPROVED** [review]
LGTM!

**Pull request merged**